lib/engine/step/g_18_co/dividend.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
This file is not yet used. It will contain necessary changes to allow for multiple jumps and potentially the payout to the corporations owning shares.